test/functional/tests/incremental_load/test_udev.py

```diff
@@ -0,0 +1,235 @@
+        elif len(caches) > 1:
+            caches_list = '\n'.join(caches)
+            TestRun.fail(f"There is more than 1 cache loaded:\n{caches_list}")
+        elif caches[0].cache_device.system_path != cache_dev.system_path:
+            TestRun.fail(f"Cache loaded on wrong device. "
+                         f"Actual: {caches[0].cache_device.system_path}, "
+                         f"expected: {cache_dev.system_path}")
+        elif caches[0].get_cache_mode() != cache_mode:
+            TestRun.fail(f"Cache did load with different cache mode. "
+                         f"Actual: {caches[0].get_cache_mode()}, expected: {cache_mode}")
+        TestRun.LOGGER.info("Cache is correctly loaded.")
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeSet([DiskType.sata, DiskType.hdd]))
+def test_neg_udev_cache_load():
+    """
+        title: Negative test for executive cache udev rules.
+        description: |
+          Verify if CAS udev rule is executed properly for cache with valid metadata and do not
+          load cache with no metadata
+        pass_criteria:
+          - No kernel error
+          - Cache with metadata is properly loaded
+          - Cache without metadata is not loaded
+          - Cores assigned to not loaded cache are not inserted to core pool after
+            plugging cache disk
+          - Cores assigned to not loaded cache are inserted to core pool after plugging core disk
+    """
+    with TestRun.step("Create init config file with two caches and two cores per each cache."):
+        cache_disk = TestRun.disks["cache"]
+        cache_disk.create_partitions([Size(1, Unit.GibiByte), Size(1, Unit.GibiByte)])
+        core_disk = TestRun.disks["core"]
+        core_disk.create_partitions([Size(2, Unit.GibiByte) for _ in range(0, 4)])
+        first_cache_core_numbers = random.sample(range(0, 4), 2)
+        init_conf = InitConfig()
+        for i in range(0, 2):
+            init_conf.add_cache(i + 1, cache_disk.partitions[i])
+        for j in range(0, 4):
+            init_conf.add_core(1 if j in first_cache_core_numbers else 2,
+                               j + 1, core_disk.partitions[j])
+        init_conf.save_config_file()
+    with TestRun.step("Start one cache and add two cores as defined in init config."):
+        cache = casadm.start_cache(cache_disk.partitions[0])
+        for i in first_cache_core_numbers:
+            cache.add_core(core_disk.partitions[i])
+    with TestRun.step("Stop cache."):
+        cache.stop()
+    with TestRun.step("Unplug and plug cache device."):
+        cache_disk.unplug()
+        cache_disk.plug()
+    with TestRun.step("Check if CAS is loaded correctly."):
+        cas_devices = casadm_parser.get_cas_devices_dict()
+        if len(cas_devices["core_pool"]) != 0:
+            TestRun.fail(f"There is wrong number of core devices in core pool. Expected: 0, actual:"
```

Comment:
I think TestRun.LOGGER.error makes more sense for most of the checks in this test